configs/a64fx_SME.yaml

```diff
@@ -294,13 +266,130 @@ Latencies:
     Execution-Latency: 6
     Execution-Throughput: 1
   11:
+    Instruction-Groups:
+    - ~1922,
+    - ~1924,
+    - ~1926,
+    - ~2359,
+    - ~2360,
+    - ~2361,
+    - ~2364,
+    - ~2365,
+    - ~2368,
+    - ~2369,
+    - ~2371,
+    - ~2390,
+    - ~2391,
+    - ~2392,
+    - ~2395,
+    - ~2396,
+    - ~2399,
+    - ~2400,
+    - ~2402,
+    - ~2445,
+    - ~2446,
+    - ~2447,
+    - ~2450,
+    - ~2451,
+    - ~2454,
+    - ~2455,
+    - ~2457,
+    - ~2470,
+    - ~2471,
+    - ~2472,
+    - ~2475,
+    - ~2476,
+    - ~2479,
+    - ~2480,
+    - ~2482,
+    - ~3627,
+    - ~3629,
+    - ~3631,
+    - ~3633,
+    - ~3644,
+    - ~3646,
+    - ~3648,
+    - ~3650,
+    - ~3709,
+    - ~3711,
+    - ~3713,
+    - ~3715,
+    - ~4306,
+    - ~4308,
+    - ~4310,
+    - ~4312,
+    - ~4326,
+    - ~4328,
+    - ~4330,
+    - ~4332,
+    - ~4372,
+    - ~4374,
+    - ~4376,
+    - ~4378,
+    - ~4468,
+    - ~4469,
+    - ~4470,
+    - ~4472,
+    - ~4474,
+    - ~4476,
+    - ~4493,
+    - ~4494,
+    - ~4495,
+    - ~4497,
+    - ~4499,
+    - ~4501,
+    - ~4511,
+    - ~4513,
+    - ~4515,
+    - ~4517,
+    - ~4519,
+    - ~4521,
+    - ~4534,
+    - ~4535,
+    - ~4536,
+    - ~4538,
+    - ~4540,
+    - ~4542,
+    - ~4594,
+    - ~4595,
+    - ~4599,
+    - ~4601,
+    - ~4603,
+    - ~4605,
+    - ~4613,
+    - ~4614,
+    - ~4618,
+    - ~4620,
+    - ~4622,
+    - ~4624,
+    - ~4633,
+    - ~4635,
+    - ~4637,
+    - ~4639,
+    - ~4641,
+    - ~4643,
+    - ~5760,
+    - ~5762,
+    - ~5764,
+    - ~5766,
+    - ~5780,
+    - ~5782,
+    - ~5784,
+    - ~5786,
+    - ~5824,
+    - ~5826,
+    - ~5828,
+    - ~5830
+    Execution-Latency: 15
+    Execution-Throughput: 1
+  12:
     Instruction-Groups:
     - SME_SIMPLE_LOGICAL
     - SME_SIMPLE_CMP
     # Same as SVE
     Execution-Latency: 4
     Execution-Throughput: 1
-  12:
+  13:
     Instruction-Groups:
     - SME_SIMPLE
     - SME_DIV_OR_SQRT
```

Comment:
Given `-SME` has been placed above with SVE, these groups should also probably be moved to be along side the SVE counterparts above.

I think it would also be better to match how SVE groups are used - i.e. don't use the catch-all of `-SME` at all, and instead place the following in the correct latency group (match to SVE):

- SME_SIMPLE
- SME_SIMPLE_ARTH
- SME_SIMPLE_ARTH_NOSHIFT
- SME_SIMPLE_LOGICAL
- SME_SIMPLE_LOGICAL_NOSHIFT
- SME_SIMPLE_CMP
- SME_SIMPLE_CVT
- SME_MUL
- SME_DIV_OR_SQRT

Reply:
So are you saying to duplicate all SVE group latency settings with an SME entry? All this config stuff was just copied over from the ModSim work so it should be what you defined previously.

Reply:
Yes. For this example implementation of SME, all latencies are identical to their SVE counterparts. I should have done this before in the ModSim work - but have only spotted it now on review

Reply:
Made the change, please double check in case I've misunderstood

Reply:
Looks good. Thanks for updating